### DISCUSSION TITLE
SD is slow

### DISCUSSION THREAD

**Discussion by darth2602** [TheLastBen/fast-stable-diffusion#1685] [General]
This is not a criticism... but I find that on Google Colab, SD is much slower than at the beginning at launch (at least 10mn to start in extended memory, normal processor) but also to calculate the images...  with a lot of blockages and needs to relaunch webui

**Comment by TheLastBen** [comment]
when you have many extensions installed it takes around 4 minutes to start, if you want speed, use my Runpod version of the notebooks

**Reply by darth2602** [reply]
ok, I only kept what I use the most, we'll see
